packages/netcdf4/test_netcdf4.py

```diff
@@ -0,0 +1,443 @@
+import pytest
+from pytest_pyodide import run_in_pyodide
+
+
+@pytest.mark.driver_timeout(60)
+@run_in_pyodide(packages=["netCDF4", "numpy"])
+def test_netCDF4_tutorial(selenium):
+    """
+    https://github.com/Unidata/netcdf4-python/blob/master/examples/tutorial.py
```

Comment:
Great, but could you replace the print statements in this test with `assert` and compare the values, so that we can check if this library works correctly?

Reply:
That's a good idea - in the end, I didn't convert each print individually but wrote an `assert_print` helper function that checks the (normalised) prints against the expected outputs.